src/main/java/de/muenchen/isi/configuration/ImportStammdatenFromFile.java

```diff
@@ -0,0 +1,158 @@
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.fileupload.FileItem;
+import org.apache.commons.fileupload.disk.DiskFileItem;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.stereotype.Component;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.multipart.commons.CommonsMultipartFile;
+
+@Component
+@Data
+@Slf4j
+public class ImportStammdatenFromFile implements CommandLineRunner {
+
+    private static final String PATH = "csv/";
+    private final StammdatenImportService stammdatenImportService;
+    private final StaedtebaulicheOrientierungswertRepository staedtebaulicheOrientierungswertRepository;
+    private final SobonOrientierungswertSozialeInfrastrukturRepository sobonOrientierungswertSozialeInfrastrukturRepository;
+    private final Boolean deferDatasourceInit;
+
+    public ImportStammdatenFromFile(
+        @Value("${spring.jpa.defer-datasource-initialization:true}") final Boolean deferDatasourceInit,
+        final StammdatenImportService stammdatenImportService,
+        final SobonOrientierungswertSozialeInfrastrukturRepository sobonOrientierungswertSozialeInfrastrukturRepository,
+        final StaedtebaulicheOrientierungswertRepository staedtebaulicheOrientierungswertRepository
+    ) {
+        this.deferDatasourceInit = deferDatasourceInit;
+        this.stammdatenImportService = stammdatenImportService;
+        this.sobonOrientierungswertSozialeInfrastrukturRepository =
+            sobonOrientierungswertSozialeInfrastrukturRepository;
+        this.staedtebaulicheOrientierungswertRepository = staedtebaulicheOrientierungswertRepository;
+    }
+
+    /**
+     * {@inheritDoc}
+     * <p>
+     * Führt den Import von Stammdaten aus den angegebenen CSV-Dateien durch.
+     */
+    @Override
+    public void run(String... args) {
+        if ((deferDatasourceInit != null && deferDatasourceInit.booleanValue())) {
+            log.info("START LOADING STAMMDATEN");
+            this.clearDatabase();
+            this.addSobonJahre(PATH);
+            log.info("FINISHED LOADING STAMMDATEN");
+        }
+    }
+
+    /**
+     * Löscht alle vorhandenen Datensätze in der Datenbank.
+     */
+    private void clearDatabase() {
+        this.sobonOrientierungswertSozialeInfrastrukturRepository.deleteAll();
+        this.staedtebaulicheOrientierungswertRepository.deleteAll();
+    }
+
+    /**
+     * Fügt die Sobon-Jahre in die Datenbank ein.
+     *
+     * @param dateipfad Der Dateipfad zum Verzeichnis der CSV-Dateien.
+     */
+    public void addSobonJahre(String dateipfad) {
```

Comment:
Alternativ können zur Reviewanmerkung https://github.com/it-at-m/isi-backend/pull/116#discussion_r1392240170 die relativen Dateipfade zu den CSV-Dateien als Listen in einer Property jeweils für die SobonSozialeinfrastruktur dund SobonOrientierungswerte vorgehalten werden.

`property.x.y: csv/SoBoNOrientierungswerteSozialeInfrastruktur2014.csv,csv/SoBoNOrientierungswerteSozialeInfrastruktur2017.csv,...`

Somit brauchen die Dateipfade nicht mehr hartkodiert werden.

Reply:
Die Properties könnnen dann analog der Property `file.mimetypes.allowed` in der Serviceklasse als Liste von Strings eingelesen werden.

Reply:
done